src/components/Convert/routes/hlpAddRemove.ts

```
@@ -145,21 +162,20 @@ const hlpAddRemoveTransactionHandler = async (
   if (!hlpInfo)
     throw new Error("hlpInfo is required for a hlpToken transaction");
 
-  const hlpManager = HlpManager__factory.connect(
-    HLP_CONTRACTS[network]?.HlpManager ?? ethers.constants.AddressZero,
+  const hlpRewardRouter = HlpRewardRouter__factory.connect(
+    HLP_CONTRACTS[network]?.HlpRewardRouter ?? ethers.constants.AddressZero,
     signer
   );
```

We should throw instead of defaulting to AddressZero

---

Good point